src/app/components/date-range-picker/index.html

```diff
@@ -0,0 +1,262 @@
+    <sky-demo-page-property
+      propertyName="label"
+    >
+      Specifies a label for the date range picker. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="calculatorIds"
+      isOptional="true"
+    >
+      Specifies IDs for the date range options to include in the list that users select from. The options specify calculators to determine the dates in date ranges. This property accepts <a stacheRouterLink="." fragment="skydaterangecalculation-properties">an array of <stache-code>SkyDateRangeCalculatorId</stache-code> values</a>. By default, this property includes <a stacheRouterLink="." fragment="skydaterangecalculation-properties">all <stache-code>SkyDateRangeCalculatorId</stache-code> values</a>.
+    </sky-demo-page-property>
+
+    <sky-demo-page-property
+      propertyName="dateFormat"
+      isOptional="true"
+      defaultValue="MM/DD/YYYY"
+    >
+      Specifies a date format for the <stache-code>sky-datepicker</stache-code> components that make up the date range picker. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="disabled"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to disable the date range picker. This property accepts <stache-code>boolean</stache-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties
+    sectionHeading="Date range calculation"
+  >
+    <p>
+      The form model for the date range picker field returns a value with a type of <stache-code>SkyDateRangeCalculation</stache-code>.
+    </p>
+    <sky-demo-page-property
+      propertyName="calculatorId"
+    >
+      Specifies the calculator that determines the dates in the date range. This property returns <a stacheRouterLink="." fragment="skydaterangecalculation-properties">a <stache-code>SkyDateRangeCalculatorId</stache-code> value</a>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="endDate"
+      isOptional="true"
+    >
+      Specifies the last date in the date range. This property returns a <stache-code>Date</stache-code> value.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="startDate"
+      isOptional="true"
+    >
+      Specifies the first date in the date range. This property returns a <stache-code>Date</stache-code> value.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties
+    sectionHeading="SkyDateRangeCalculatorId properties"
+  >
+    <sky-demo-page-property
+      propertyName="After"
+    >
+      Enables users to select a start date with no end date.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="AnyTime"
+    >
+      Selects no dates and considers all dates within the date range. This is the default selection.
```

Comment:
Should `This is the default selection.` have some visual importance to stand out in this large crowd? Bold, highlight, etc... ?